gitea.rb

```diff
@@ -43,7 +43,7 @@ def install
     when build.devel?
       bin.install "#{buildpath}/gitea-master-darwin-10.6-amd64" => "gitea"
     else
-      bin.install "#{buildpath}/gitea-1.1.4-darwin-10.6-amd64" => "gitea"
+      bin.install "#{buildpath}/gitea-#{version}-darwin-10.6-amd64" => "gitea"
```

Comment:
Maybe line 43 -> line 45 could be removed since `version` is also defined on `devel`.